### DISCUSSION TITLE
希望可以增加选项：不显示动态下方的热门评论

### DISCUSSION THREAD

**Discussion by asterho** [the1812/Bilibili-Evolved#3659] [功能建议 / Ideas]
<img width="646" alt="Screen Shot 2022-09-04 at 19 05 00" src="https://user-images.githubusercontent.com/21488882/188308157-1bbd200c-eac9-4f08-ae0e-1a6164b23b96.png">
<img width="646" alt="Screen Shot 2022-09-04 at 19 02 39" src="https://user-images.githubusercontent.com/21488882/188308163-d5f8b196-ec8b-4575-a838-bec9044ded61.png">

**Comment by Tinhone** [comment]
js循环删除
```
for(let i1 of document.querySelectorAll(".bili-dyn-item__interaction")){
    i1.remove()
}
```
或者弄个样式
```
.bili-dyn-item__interaction{
    display: none;
}
```
不知道作者大大会不会加 ( ﾟ∀。)

**Reply by asterho** [reply]
感谢！

**Comment by the1812** [comment]
重复 #3322
